lib/scorpio/resource_base.rb

```diff
@@ -349,7 +349,7 @@ def call_operation(operation, call_params: nil, model_attributes: nil)
         if operation.responses
           _, operation_response = operation.responses.detect { |k, v| k.to_s == response.status.to_s }
           operation_response ||= operation.responses['default']
-          response_schema = operation_response.schema if operation_response
+          response_schema = operation_response['schema'] if operation_response
```

Comment:
downside of naming the SchemaInstance's schema as #schema is collisions with property accessors called schema such as this one. I've decided that's acceptable.